akka-persistence/src/test/scala/akka/persistence/SnapshotRecoveryLocalStoreSpec.scala

```diff
@@ -22,9 +22,10 @@ object SnapshotRecoveryLocalStoreSpec {
   }
 
   class LoadSnapshotTestPersistentActor(name: String, probe: ActorRef) extends NamedPersistentActor(name)
-    with TurnOffRecoverOnStart
     with ActorLogging {
 
+    override def recovery = Recovery(toSequenceNr = 0)
```

Comment:
Yes, it's a bit weird in the test but should not impact real apps. Looks good then?